lang/cs/Org.Apache.REEF.Examples.HelloREEF/App.config

```diff
@@ -18,7 +18,15 @@ specific language governing permissions and limitations
 under the License.
 -->
 <configuration>
-    <startup> 
-        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5" />
-    </startup>
+  <startup>
+    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5" />
+  </startup>
+  <runtime>
```

Comment:
I couldn't find such way. But, as far as I understand, if user executable doesn't use Avro or Json packages directly, only via REEF dlls, they don't need any action. When we run tests, we use only REEF dlls, we don't have app.config for tester executable, and correct version of package propagates.